Riot/Modules/Room/Members/Detail/RoomMemberDetailsViewController.m

```diff
@@ -461,7 +461,7 @@ - (void)presentCompleteSecurity
 - (void)showRoomWithId:(NSString*)roomId
 {
 //    Analytics.shared.viewRoomTrigger = AnalyticsViewRoomTriggerRoomMemberDetail;
```

Comment:
I think we can keep Analytics from Element, this is disabled by default in Tchap